CHANGELOG/CHANGELOG-1.26.md

```diff
@@ -486,7 +486,8 @@ name | architectures
   - [KEP]: <link>
   - [Usage]: <link>
   - [Other doc]: <link>
-  --> ([#86139](https://github.com/kubernetes/kubernetes/pull/86139), [@jasimmons](https://github.com/jasimmons)) [SIG API Machinery, Apps, Architecture, Auth, Autoscaling, CLI, Contributor Experience, Instrumentation, Network, Node, Release, Scheduling, Storage and Testing]
+  -->
+- Container `preStop` and `postStart` lifecycle handlers using `httpGet` now honor the specified `scheme` and `headers` fields. This enables setting custom headers and changing the scheme to `HTTPS`, consistent with container startup/readiness/liveness probe capabilities. Lifecycle handlers configured with scheme `HTTPS` that encounter errors indicating the endpoint is actually using HTTP fall back to making the request over HTTP for compatibility with previous releases.([#86139](https://github.com/kubernetes/kubernetes/pull/86139), [@jasimmons](https://github.com/jasimmons)) [SIG API Machinery, Apps, Architecture, Auth, Autoscaling, CLI, Contributor Experience, Instrumentation, Network, Node, Release, Scheduling, Storage and Testing]
```

Comment:
Release notes tags with `Action Required` usually get pulled up under `(No, really, you MUST read this before you upgrade)`  section of the document in Change Logs.  I think it would be better to pull this one under that section ? Please check and see if that makes sense.

Reply:
xref: https://github.com/kubernetes/kubernetes/blob/a0798987e13c51c216937fc2b56a3e0d1d002875/CHANGELOG/CHANGELOG-1.26.md#no-really-you-must-read-this-before-you-upgrade with PR https://github.com/kubernetes/kubernetes/pull/111973